src/types/ExchangeApi.ts

```diff
@@ -1,10 +0,0 @@
-import BigNumber from 'bignumber.js';
-
-export default interface ExchangeApi {
```

Comment:
NIT: since we removed this interface, I'd rename the abstract class to `ExchangeApi`

Reply:
Done

Reply:
Done.